toolchains/install_bootstrap/android/tool_redirect_rules.yaml

```diff
@@ -0,0 +1,30 @@
+bin: /toolchain_root/bin
+
+overwrite: []
+
+replacements:
+  cc:  arm-linux-androideabi-gcc
```

Comment:
Yes. However, only the android toolchain prefixes all its tools like this (musl doesn't).

It can work either way. The way I have it now, `bin` _must_ be a directory, so in the code I use `os.path.join` to concatenate the `bin` directory with the replacement tool. If it was a prefix (not necessarily a directory), I would need to use string concatenation instead (i.e. `"%s%s" % (prefix, overwrite)`). I don't mind either way, up to you which way you think is cleaner